app/livechat/client/views/app/tabbar/customerChatHistory.html

```diff
@@ -0,0 +1,73 @@
+<template name='customerChatHistory'>
+	<div class="content">
+		<div class="list-view">
+			<div>
+			<div class="active-pink-4 mb-4 inner-addon right-addon">
+				<i class="glyphicon glyphicon-search icon-search"></i>
+				<input type="text" id="searchInput" class="" placeholder="Search" aria-label="Search">
+			</div>
+		</div>
+            {{#if isAllChat}}
+			<div class="vistior-history-block">
+				<ul id='allist'>
+				{{#if isSearching}}
+					{{#if isfound }}
+					{{#each searchResults}} 
```

Comment:
You don't need a `searchResults` method, you can always use `previousChats`.
You just need to pass the search term to the endpoint that provides the `previousChats`.
```suggestion
    		{{#each room in previousChats}} {{> chatRoomHistoryItem room }} {{/each}}
```

Reply:
Created a template called chatRoomHistoryItem